nsxt/resource_nsxt_policy_group_test.go

```diff
@@ -381,6 +381,85 @@ func TestAccResourceNsxtPolicyGroup_multipleNestedCriteria(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtPolicyGroup_identityGroup(t *testing.T) {
+	name := fmt.Sprintf("test-nsx-policy-group-identity-group")
+	testResourceName := "nsxt_policy_group.test"
+	updatedName := fmt.Sprintf("%s-update", name)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t); testAccOnlyLocalManager(t) },
```

Comment:
Is there a reason for LM only?

Reply:
Yes, identity group is not supported on GM.